picard/ui/options/plugins.py

```diff
@@ -33,10 +34,6 @@
 from picard.ui.ui_options_plugins import Ui_PluginsOptionsPage
 
 
-def cmp_plugins(a, b):
```

Comment:
The benefit of the explicitly defined function is that there's only one place it's defined, instead of twice.